### DISCUSSION TITLE
Multiple Pools on Same Network

### DISCUSSION THREAD

**Discussion by af85** [Chia-Network/chia-blockchain#693] [General]
I have two different pools running on the same network. Basically two different servers that have their own unique wallets. One server is new and was upgraded to b19 and has 151 k32's, and it has TXCH being farmed regularly. The other server was just upgraded from b17 to b20, has 1600 k32's, and has 0 TXCH farmed (it had several TXCH on b17).

This b20 server was being awarded TXCH quite regularly before the upgrade to b20, and it's been running for a day and half now with no rewards. I just wanted to make sure there isn't an issue with b20, or that there isn't an issue with having two unique pools running on the same network. Is there anything I can run to confirm there are no issues?

(venv) [user@server]$ chia wallet
Balances
Wallet ID 1 type STANDARD_WALLET Chia Wallet
   -Confirmed: 0.0
   -Unconfirmed: 0.0
   -Spendable: 0.0
   -Frozen: 0.0
   -Pending change: 0.0

**Comment by hoffmang9** [comment]
You most likely have both of them running uPnP and they are competing and confusing each other. Set `enable_upnp: false` in `~/.chia/VERSION/conf/config.yaml` on all but one node on the same network.

**Reply by btalachian** [reply]
> You most likely have both of them running uPnP and they are competing and confusing each other. Set `enable_upnp: false` in `~/.chia/VERSION/conf/config.yaml` on all but one node on the same network.

Can I use 2 Fullnode in different public IPs (different networks) with ONE wallet address?

**Reply by N0rdmann** [reply]
**would like to know the same**, what happens when running the same (copied over) full gui instance including keys with full_node, farmer, wallet service on another pc with another external ip address

**Reply by NetrunnerAT** [reply]
i use 4 "full node only" in my DMZ. This work if any full node use a other TCP Port. Normal Port forwarding is enough. Behind the DMZ are 4 "Farmer Node" with some Harvesters connected. If one Full node lost connection ... the other 3 stay synced ;-)

**Comment by JordyBaylac** [comment]
what happen with the port forwarding in this case, given is targeting one node, will the other be left behind?

**Reply by rawbonesfck** [reply]
It seems so.  What i'm doing currently is trying to block 8444 on the 'left behind' node and connecting it to the 'full node' via connect option.  I also edited the config file to reflect 'harvester' and 'full node' of the 'left behind' to connect to the main node.
no idea if this is going to work.
